.github/ISSUE_TEMPLATE/general-issues.md

```diff
@@ -0,0 +1,41 @@
+---
+name: General Issues
+about: Please use this template to submit issues for review
+title: ''
+labels: needs-triage
+assignees: ''
+
+---
+
+**Note: for support questions, please first reference our [documentation](https://docs.aws.amazon.com/cdk/api/latest), then use [Stackoverflow](https://stackoverflow.com/questions/ask?tags=aws-cdk)**. This repository's issues are intended for feature requests and bug reports.
+
+* **I'm submitting a ...**
+  - [ ] :beetle: bug report
+  - [ ] :rocket: feature request
+  - [ ] :books: construct library gap
+  - [ ] :phone: security issue or vulnerability => Please see [policy](https://github.com/awslabs/aws-cdk/security/policy)
+  - [ ] :question: support request => Please see note at the top of this template.
+
+
+* **What is the current behavior?**
+*If the current behavior is a :beetle:bug:beetle:: Please provide the steps to reproduce*
+
```

Comment:
What's the difference between general-issues and issue_tempalte

Reply:
Old vs new format. Basically, in order to do autolabeling, it requires the new format. The issue_template will now also populate the correct formatting into the "open regular issue" link on the bottom of the "New Issue" screen. This method doesnt allow for autolabeling, so I added a newer format without sacrificing the old one